flutter/windows/runner/Runner.rc

```diff
@@ -89,11 +89,11 @@ BEGIN
     BEGIN
         BLOCK "040904e4"
         BEGIN
-            VALUE "CompanyName", "com.xored" "\0"
+            VALUE "CompanyName", "org.mlcommons" "\0"
```

Comment:
I guess that here we want to place the actual company name and not just `org.mlcommons`.
I guess this would be `MLCommons Association`, or something.
Same goes for the `LegalCopyright` field.